### ISSUE TITLE
Set padding on xml lead to unexpected behaviour

### ISSUE THREAD

**Issue by nofurylion** [MikeOrtiz/TouchImageView#209]
**How to reproduce** 
1. On xml
```

<?xml version="1.0" encoding="utf-8"?>
<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
    xmlns:app="http://schemas.android.com/apk/res-auto"
    xmlns:tools="http://schemas.android.com/tools"
    android:layout_width="match_parent"
    android:layout_height="match_parent"
    android:layout_margin="8dp"
    android:gravity="center"
    android:orientation="vertical"
    tools:context=".MainActivity">


    <LinearLayout
        android:layout_width="wrap_content"
        android:layout_height="wrap_content">

        <com.ortiz.touchview.TouchImageView
            android:id="@+id/myIV"
            android:layout_width="wrap_content"
            android:layout_height="wrap_content"
            android:background="@drawable/border"
            android:src="@drawable/messi5"
            android:padding="16dp" />
    </LinearLayout>
</LinearLayout>

```

The result is : 
![device-2019-09-07-070514](https://user-images.githubusercontent.com/23353377/64475934-39c3df80-d1b3-11e9-850c-7c14836fb824.png)

Expected behaviour(taken from normal ImageView with scaletype fitXY)
![device-2019-09-07-070848](https://user-images.githubusercontent.com/23353377/64475988-b5259100-d1b3-11e9-882d-94bf9c083578.png)

Same issue [here](https://github.com/MikeOrtiz/TouchImageView/pull/62) and [here](https://github.com/MikeOrtiz/TouchImageView/issues/102#issue-41461353)

What exactly happen?

**Comment by mahditakrim** [comment]
Hi @nofurylion ,
Could you please explain this issue a little bit more ?
And may I work on it ?

**Comment by hannesa2** [comment]
> And may I work on it ?

@nofurylion of course, and yes please !

**Comment by Sola85** [comment]
I have the same issue. Setting a padding != 0 results in the padding being applied only to the upper and left edges of the image, i.e. the image is not centered anymore like it would be in a normal ImageView.

**Comment by hannesa2** [comment]
I made a pull request for this issue, just to recap #286
Guys, when you want to have a faster solution, make a pull request showing the issue. Then it's much easier to understand your issue. Without sample, I can only guess what exactly you want. 

First, this is obsolete @nofurylion 

    <LinearLayout
        android:layout_width="wrap_content"
        android:layout_height="wrap_content">

**Comment by hannesa2** [comment]
Third this is my way to solve it 4b08ac1

![image](https://user-images.githubusercontent.com/3314607/96550919-cf380b00-12b1-11eb-8f5d-00d18b65efae.png)

**Comment by hannesa2** [comment]
I close it, but when it's not your issue, please re-open this issue again and **update the sample pull request** with your **exact** issue

**Comment by hannesa2** [comment]
It's probably solved with #655
